Gruntfile.js

```diff
@@ -0,0 +1,87 @@
+module.exports = function (grunt) {
+    grunt.initConfig({
+        sass: {
+            dist: {
+                options: {
+                    style: 'expanded'
+                },
+                files: {
+                    'src/Frontend/styles/custom.css': 'src/Frontend/styles/custom.scss',
+                    'web/build/temp/foundation.css': 'node_modules/zurb-foundation-5/scss/foundation.scss',
+                    'web/build/temp/normalize.css': 'node_modules/zurb-foundation-5/scss/normalize.scss'
+                }
+            }
+        },
+        concat: {
+            options: {
+                separator: ';'
+            },
+            dist: {
+                src: [
+                    'web/build/temp/foundation.css',
+                    'web/build/temp/normalize.css',
+                    'src/Frontend/styles/custom.css'
+                ],
+                dest: 'web/build/bundle.css'
+            }
+        },
+        browserify: {
+            client: {
+                src: ['src/Frontend/**/*.js'],
```

Comment:
you have to ignore all files of the test directory in order to prevent the compilation of these: http://stackoverflow.com/questions/29034967/grunt-browserify-ignore-option